dynamic_rest/viewsets.py

```diff
@@ -384,6 +384,14 @@ def list_related(self, request, pk=None, field_name=None):
 
         return Response(serializer.data)
 
+    def get_extra_filters(self, request):
+        # Override this method to enable addition of extra filters
+        # (i.e., a Q()) so custom filters can be added to the queryset without
+        # running into https://code.djangoproject.com/ticket/18437
+        # which, without this, would mean that filters added to the queryset
+        # after this is called may not behave as expected.
+        return None
```

Comment:
Perhaps better to return `Q()` instead here and add the filters in every time. I think I was hesitant to do this because of this https://code.djangoproject.com/ticket/24279, which according to `git tag --contains afe0bb7b13bb8dc4370f32225238012c873b0ee3`, was only introduced into Django in 1.9.

Reply:
I think returning None is fine.